challenge_7/python/popcorn_anachronism/src/challenge_7.py

```diff
@@ -0,0 +1,11 @@
+test1 = [0,1,2,4]           #answer: 3
+test2 = [9,6,7,8,5,4,1,3,0] #answer: 2
+test3 = list(range(100000)) #answer: 99 
+test3.remove(99)
+
+def find_missing(array):
+    return sum(range(len(array)+1))-sum(array)
```

Comment:
The solution should use O(1) space.  Pythons `range()` function will create a list; because of this you are using O(N) space.  There is a formula that will calculate the sum for you.